### PR TITLE
Feat : Landing page

### PR DESCRIPTION
**Pull request by sharathnair9999**
> Created Folder Structure

> Created Landing Page

> Added Routes to Products , Product Pages

> Responsiveness

ecom-landing-page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import Address from "./components/Address/Address";
+import Orders from "./components/Orders/Orders";
+import Order from "./components/Orders/Order";
+import Mockman from "mockman-js";
+
+function App() {
+  const { initialState } = userDetails();
+  return (
+    <>
+      <div>
```
Comment (spraveenofficial) [inline src/App.js:25]: You can remove this div

**Review by spraveenofficial — APPROVED** [review]

**Review by SukanyaSen530 — APPROVED** [review]
Could have separated the PR for initial setup and landing page. Overall good folder structure and liked the navbar mobile responsive animation.  🚀

**Review by ArjunGTX — COMMENTED** [review]
- Both React and Mockbee boiler plateing can be removed
- Try to keep the PR specific

File: src/pages/Profile.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import { Link, Outlet, useParams } from "react-router-dom";
+import { userDetails } from "../contexts/user-context";
+
+const Profile = () => {
+  const {initialState} = userDetails()
+  const {id} = initialState
+  return (
+    <div style={{display:"flex" , justifyContent:'flex-start', alignItems:"center", gap:"2rem"}}>
+
+    <div style={{borderRight:"1px solid gray", height:"100%"}}>
```
[the remark this answers is not in the document]
Comment (sharathnair9999) [reply src/pages/Profile.js:11]: Yes. That component i am not using yet. Will make the suggested changes

**Pull request merged**